dracut/30coreos-installer/parse-coreos.sh

```diff
@@ -28,7 +28,7 @@ fi
 # Parse all args (other than rd.neednet) and persist those into /tmp/networking_opts
 # List from https://www.mankier.com/7/dracut.cmdline#Description-Network
```

Comment:
can we mention here that not all of them come from dracut and some come from kernel: https://github.com/torvalds/linux/blob/master/Documentation/networking/ipv6.txt

maybe we even make a new array below for ones from the kernel and then we have `DRACUT_NET_ARGS` and `KERNEL_NET_ARGS` that we then combine on the for loop line.

Reply:
separate arrays sounds like a good idea!